repose-aggregator/components/services/atom-feed-service/atom-feed-service-impl/src/main/scala/org/openrepose/nodeservice/atomfeed/impl/AtomEntryStreamBuilder.scala

```diff
@@ -112,6 +132,10 @@ object AtomEntryStreamBuilder {
 
   case class UnrecoverableIOException(cause: Throwable) extends RuntimeException(cause)
 
+  case class ServerResponseException(message: String) extends Exception(message)
+
+  object ClientErrorException extends Exception
+
   object AuthenticationException extends Exception
```

Comment:
Not sure how i missed this the first time, but there are very few times when an exception has absolutely no arguments to a constructor.